src/public/sass/ITCSS/06_components/pages/privacy-policy/_privacy-policy.scss

```
@@ -0,0 +1,25 @@
+@use '../../../01_settings/global' as *;
+@use '../../../01_settings/colors' as *;
+@use '../../../01_settings/fonts' as *;
+@use '../../../02_tools/breakpoints' as *;
+@use '../../../02_tools/mixins' as *;
+
+.privacy-policy {
+  &__heading {
+    color: $color-aqua-blue;
+  }
+
+  &__text {
+    color: $color-gray-50;
+    font-size: 1.2rem;
```

The `font-size:1.2rem` make the font a little big. I think 1rem is good
```suggestion
    font-size: 1rem;
```